### DISCUSSION TITLE
It is not working to Adding an Extras Keymaps.

### DISCUSSION THREAD

**Discussion by nolongerhesitate** [LazyVim/LazyVim#4949] [Q&A]
Hi, I've installed the codeium by **Lazy Extras**. I want to add a keymap to create a codeium chat but it is not working. I can't see it on the leader menu.

This is my config file(`lua/config/codeium.lua`):
``` 
return {
  "Exafunction/codeium.nvim",
  keys = {
    { "<leader>xc", "<cmd>:Codeium Chat<cr>", desc = "Codeium Chat" },
  },
}
```

Am I missing something?

**Comment by dpetka2001** [comment]
This exact example works perfectly fine for me in a LazyVim default installation. I can see `<leader>xc` on the leader menu.

**Reply by nolongerhesitate** [reply]
Well, I have solved this problem. I should put the `codeium.lua` in plugin folder rather than config folder.
Happy😄.
